### PR TITLE
GetAvailableSpace(block) now returns error

### PR DESCRIPTION
**Pull request by awels**
Modified function that gets the size of a block device/available to return error as well as -1, so we
can distinguish the path not existing from the binary not existing in case the container doesn't have
the required binaries.

Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
GetAvailableSpace(block) did not return an error, it would return -1 in case of a failure, but that left us unable to distinguish between the file/device not existing vs other failures. We now properly return (and report) the error in case of failures.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
BugFix: GetAvailableSpace now properly returns error on failures.
```

### CODE REVIEW

File: pkg/importer/data-processor_test.go [hunk 1]
```diff
@@ -389,8 +389,8 @@ var _ = Describe("Resize", func() {
 	})
 
 	It("Should return same value as replaced function", func() {
-		replaceAvailableSpaceBlockFunc(func(dataDir string) int64 {
-			return int64(100000)
+		replaceAvailableSpaceBlockFunc(func(dataDir string) (int64, error) {
+			return int64(100000), nil
 		}, func() {
```
Comment (aglitke) [inline pkg/importer/data-processor_test.go:394]: We should add a test where the AvailableSpaceBlockFunc returns an error.

Comment (awels) [reply pkg/importer/data-processor_test.go:394]: So in this particular case we ignore errors from the function and log them. Then continue on base on the result. So adding the test would not add much value.

File: pkg/importer/data-processor_test.go [hunk 2]
```diff
@@ -389,14 +389,26 @@ var _ = Describe("Resize", func() {
...
 			mdp := &MockDataProvider{}
 			dp := NewDataProcessor(mdp, "dest", "dataDir", "scratchDataDir", "")
 			Expect(int64(100000)).To(Equal(dp.calculateTargetSize()))
 		})
 	})
+
+	It("Should fail is calculate size returns failure", func() {
```
Comment (aglitke) [inline pkg/importer/data-processor_test.go:401]: Should fail if

**Pull request merged**